src/main/java/knightminer/simplytea/item/CocoaItem.java

```diff
@@ -29,16 +30,17 @@ public CocoaItem(Properties props) {
 
 	@Override
 	public ItemStack finishUsingItem(ItemStack stack, Level worldIn, LivingEntity living) {
+		boolean isMilk = stack.is(Registration.cup_frothed);
```

Comment:
I don't like having to special case this. Lets instead make the `CocoaItem` constructor take in a `CocoaDrink` parameter, then use that field instead of a static reference through config.

Reply:
Since the CocoaDrink config was already being passed in as FoodProperties, I used that in CocoaItem instead (similar to how it was already used in TeaCupItem).